### ISSUE TITLE
Refresh time not take into account in Web Server mode

### ISSUE THREAD

**Issue by nicolargo** [nicolargo/glances#784]
I was using glances -t 15 in order to force it to update only once every 15 seconds. This worked great, and kept the CPU usage of Glances down! Now I'm trying to use it in web server mode, but -t doesn't seem to do anything. No matter what I do, it always updates once every 3 seconds and is using 13% CPU to do it!

Is there any way to make the webserver mode update less frequently?

Source:  https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/glances-users/CpiI6ISyCsI/PB3WJWs0BwAJ

@nclsHart / @notFloran : can you have a look, refresh time (-t option) does not work as expected in (-w) mode.

**Comment by notFloran** [comment]
Temporary workaround : http://0.0.0.0:61208/?refresh_time=15

The WebUI will wait 15s before loading the data from the API, I will fix that.

For the refresh time (-t option) that does not work as expected, it is a more general problem of parameters passed as arguments. 
We must find a way to pass the values to the web interface

**Comment by nicolargo** [comment]
@notFloran  Any idea why the parameter are not passed to the Web UI ?

**Comment by nicolargo** [comment]
@notFloran 
If i add a log on this line: https://github.com/nicolargo/glances/blob/develop/glances/outputs/glances_bottle.py#L119
The refresh_time var is set to the -t <val> value...
Dor not understatnd why is it not take into account in the Web UI...

**Comment by notFloran** [comment]
The arguments are not taken into account in the WebUI because the WebUI access Glances data only by making call to the API from the javascript. The index is a simple html file without any logic.

The refresh_time variable in *_index* is not used at all, I think it comes from the old interface.

We have two solutions to access refresh_time and others arguments from the WebUI : 

* Serialize arguments in json and add them to the index in a <script> (move from a static file to a template) so we can access the values from javascript
* Add a new endpoint in the API to returns arguments

What do you think @nclsHart @nicolargo  ?

**Comment by nicolargo** [comment]
I prefer the second solution. It allows third party software to access to the command line argument through the API.

I am going to add a new endpoint to the API. After that, you can implement the features in the WebUI.

**Comment by nicolargo** [comment]
Done in the latest commit:

```
# curl http://0.0.0.0:61208/api/2/args
{"full_quicklook": false, "snmp_version": "2c", "network_cumul": false, "help_tag": false, "snmp_port": 161, "export_csv": null, "export_statsd": false, "network_sum": false, "enable_process_extended": false, "process_short_name": true, "snmp_community": "public", "disable_sensors": false, "port": 61208, "disable_mem": false, "fahrenheit": false, "disable_swap": false, "export_opentsdb": false, "percpu": false, "password_prompt": false, "snmp_auth": "password", "disable_left_sidebar": false, "conf_file": null, "disable_process": false, "disable_quicklook": false, "disable_autodiscover": false, "snmp_force": false, "disable_irix": false, "disable_ip": false, "disable_docker": false, "username": "glances", "disable_raid": false, "export_influxdb": false, "diskio_iops": false, "disable_hddtemp": true, "disable_bg": false, "disable_log": false, "snmp_user": "private", "webserver": true, "disable_top": false, "disable_fs": false, "export_rabbitmq": false, "byte": false, "password": "", "path_history": "/tmp", "disable_bold": false, "process_tree": false, "fs_free_space": false, "enable_history": false, "no_kernel_threads": false, "diskio_show_ramfs": false, "disable_load": false, "quiet": false, "disable_folder": false, "process_filter": null, "theme_white": false, "client": null, "disable_cpu": false, "time": 5, "debug": false, "bind_address": "0.0.0.0", "disable_diskio": false, "server": false, "disable_network": false, "browser": false}
```

or to only get one argument:

```
# curl http://0.0.0.0:61208/api/2/args/time
5
```

**Comment by notFloran** [comment]
Ok thanks, I'll implement it in the WebUI.

**Comment by notFloran** [comment]
Fix by #789.

To change the refresh time : ```glances -w -t 30``` or http://0.0.0.0:61208/30
